modules/python/nitf/source/nitro.i

```diff
@@ -156,6 +156,17 @@
     }
 }
 
+%typemap(in) nrt_IOHandle{
+    if ($input)
+    {
+        $1 = (nrt_IOHandle) PyInt_AsLong($input);
```

Comment:
Does this work ok when you happen to get a pointer that's > 32 bits on Windows?  I know we had to take care and use `long long` in SIX in some spots.  Do you need to use `PyInt_AsSsize_t()`?